mis_builder/models/mis_report_instance.py

```diff
@@ -376,6 +380,8 @@ def _default_company(self):
     date_from = fields.Date(string="From")
     date_to = fields.Date(string="To")
     temporary = fields.Boolean(default=False)
+    account_analytic_id = fields.Many2one(
+        comodel_name='account.analytic.account', string='Analytic Account')
```

Comment:
@adrienpeiffer there is a mix of `analytic_account_id` and `account_analytic_id`.

Could we name the field `analytic_account_id` as in `mis_builder_budget` with an oldname attribute?